menas/src/main/resources/application.properties.template

```diff
@@ -68,9 +68,10 @@ menas.environment=Dev
 
 menas.spark.master=local[1]
 
-# The template for Lineage (Spline) display. If empty Lineage button is not shown.
+# The api gateway for Lineage (Spline) display. If empty Lineage button is not shown.
 # Avoiding the http(s) protocol will cause that the same protocol will be used as for Menas itself, avoiding issues
-spline.urlTemplate=//localhost:8080/spline/dataset/lineage/_search?path=%s&application_id=%s
+menas.lineage.readApiUrl=//localhost:8080/spline-gw/consumer
+lineageService=//localhost:8080/spline-ui/app/lineage-overview?executionEventId=%s
```

Comment:
```suggestion
lineageService.url=//localhost:8080/spline-ui/app/lineage-overview?executionEventId=%s
```
The name `lineageService` is used throughout but actually refers to a URL, it might make it clearer if all the variables in the code were renamed to `lineageServiceUrl` as well.

Reply:
Sorry, I should have marked this as work in progress. This is actually removed in of the the further branches originating from this one.